src/main/kotlin/com/wire/apps/polls/setup/logging/JsonLoggingLayout.kt

```diff
@@ -29,10 +29,6 @@ class JsonLoggingLayout : LayoutBase<ILoggingEvent>() {
             "thread_name" to event.threadName
         )
 
-        finalMap.includeMdc(event, INFRA_REQUEST)
```

Comment:
You can probably remove all this file now and just use `LogstashEncoder` in logback.xml.
Check the SDK for the code and the dependency